cli/tools/bundle/provider.rs

```diff
@@ -111,30 +112,148 @@ fn process_output_files(
   Ok(())
 }
 
+pub struct CliBundleResolver {
+  plugin_handler: Arc<super::DenoPluginHandler>,
+}
+
+impl CliBundleResolver {
+  pub fn new(plugin_handler: Arc<super::DenoPluginHandler>) -> Self {
+    Self { plugin_handler }
+  }
+}
+
+fn convert_import_kind(
+  kind: deno_bundle_runtime::ImportKind,
+) -> esbuild_client::protocol::ImportKind {
+  match kind {
+    deno_bundle_runtime::ImportKind::ImportStatement => {
+      esbuild_client::protocol::ImportKind::ImportStatement
```

Comment:
Is there a reason we have both `deno_bundle_runtime::ImportKind` and `esbuild_client::protocol::ImportKind`?